front/src/pods/table-player/table-player.component.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import * as classes from './table-player.styles';
+import { PlayerComponent } from 'pods/player/player.component';
+import { Player, VoteResult } from '../master/master.vm';
+import Typography from '@material-ui/core/Typography';
+import CheckIcon from '@material-ui/icons/Check';
+import CloseIcon from '@material-ui/icons/Close';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+interface DataPlayers {
+  nickname: string;
+  voted?: boolean;
+  vote?: string;
+}
+
+interface Props {
+  playersCollection: DataPlayers[];
+}
+
+export const TablePlayerComponent: React.FC<Props> = (props: Props) => {
```

Comment:
TablePlayerComponent should be placed in _common-app/components_

- It has no containers.
- It just receives clear the props from the parent.
- It can be reused in several pages in the app, but has no sense trying to reuse it in other applications.